docs/team-agreements.md

```diff
@@ -0,0 +1,26 @@
+# Development and Version Control
+
+- Branch naming policy is as follows: [issue#]-brief-description. In the event that an issue does not already exist for the changes being made in a branch, one should be made using the repos' issue templates.
```

Comment:
Why do we want branch naming policies? I have no strong opinion here, but the rationale behind the agreement should be documented

Reply:
This is from Connectivity, and I don't know their rationale, but I personally like having the issue number in the branch name because: 1) enforces our team agreement about always having an issue for every PR, and 2) clarity--I sometimes mix up branches if they're only named with description and multiple branches have similar descriptions